### PR TITLE
WIP: fix JAMM

### PR DESCRIPTION
**Pull request by dmaticzka**
* [X] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [X] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [X] This PR updates an existing recipe.
* [X] This PR does something else (explain below).

Current installations of JAMM segfault. I've added the official test to check out what's going on and try to fix this. Sould be some dependency, the test is working with manual JAMM install and current R from RCRAN.

### CODE REVIEW

File: recipes/jamm/meta.yaml [hunk 1]
```diff
@@ -10,21 +10,28 @@ source:
...
     - r-signal
     - r-mclust >=4
+    - curl
 
 test:
   commands:
     - JAMM.sh | grep JAMM > /dev/null
     - SignalGenerator.sh | grep JAMM > /dev/null
+    - mkdir testdata
+    - curl bioinf.uni-freiburg.de/~maticzkd/jamm_testdata/ctcfRep1.chr21.bed > testdata/ctcfRep1.chr21.bed
```
Comment (bgruening) [inline recipes/jamm/meta.yaml]: can you just create a 3 line BED file on the fly?

**Comment by bgruening** [comment]
Seems you are passing the conda tests, but not the Docker tests. So no segault here. If you need tiny test files you could also put them into the package and fetch them during build.sh.
Thanks.

**Comment by dmaticzka** [comment]
Thanks for having a look. Yes, the test case is not optimal yet. But it's temporary for now.

So far, R doesn't even start because of missing or incompatible libreadline. No chance for a segfault there :) Let's see if enforcing conda-forge r-base gets this further.

**Comment by dmaticzka** [comment]
That's it. Should fix the mulled-test, osx and hopefully use the right dependencies after install.

**Pull request merged**